Firestore/Source/Util/FSTDispatchQueue.mm

```diff
@@ -16,6 +16,8 @@
 
 #import <Foundation/Foundation.h>
 
+#import <atomic>
```

Comment:
I'm not sure if `#import` or `#include` is preferred for C++ standard library headers in Objective-C++ code.